### PR TITLE
Switcher

### PR DESCRIPTION
**Pull request by Kanybekov26**

### CODE REVIEW

**Comment by mirlnovna** [comment]
App.js must be cleaned

File: src/components/UI/AcceptBar.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import Switch from '@mui/material/Switch'
+import { FormControlLabel, Grid } from '@mui/material'
+import styled from '@emotion/styled'
+
```
Comment (songhee24) [inline src/components/UI/AcceptBar.jsx]: Screenshot 2023-04-06 at 13 29 55
why file and component name are different ?

File: src/components/UI/AcceptBar.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+const StyledContainer = styled('div')`
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   box-sizing: border-box;
+   padding-left: 20px;
+   padding-right: 18px;
+   width: 1140px;
```
Comment (songhee24) [inline src/components/UI/AcceptBar.jsx]: removed fixed width

File: src/components/UI/AcceptBar.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import Switch from '@mui/material/Switch'
+import { FormControlLabel, Grid } from '@mui/material'
+import styled from '@emotion/styled'
+
+export const SwitchesSize = ({ checked, onChange, ...rest }) => {
+   return (
+      <StyledContainer checked={checked}>
+         <p>ответов</p>
```
Comment (songhee24) [inline src/components/UI/AcceptBar.jsx]: {count of answers} ответов 
Screenshot 2023-04-06 at 13 32 23
must pass as props

**Review by songhee24 — CHANGES_REQUESTED** [review]

**Review by songhee24 — APPROVED** [review]

**Pull request merged**